src/main/resources/successfulApplying.fxml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.layout.Pane?>
+<?import javafx.scene.text.Font?>
+<?import javafx.scene.text.Text?>
+
+<Pane maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="400.0"
+      prefWidth="600.0" xmlns="http://javafx.com/javafx/15.0.1" xmlns:fx="http://javafx.com/fxml/1"
+      fx:controller="employment.system.controllers.SuccessfulRegistrationController">
+    <children>
+        <Text layoutX="27.0" layoutY="213.0" strokeType="OUTSIDE" strokeWidth="0.0" text="You Were Successfully Applied"
```

Comment:
text="You Were Successfully Applied"  - does not sound right.

Reply:
I will do that in the next story ( ESFI-27 - Refactoring) because we decided to modify some issues there.